.circleci/config.yml

```diff
@@ -0,0 +1,72 @@
+version: 2.1
+
+references:
+  images:
+    go: &GOLANG_IMAGE circleci/golang:1.12.5
```

Comment:
Do we pin to a specific version genrally in other libs? I can see why we do for binaries where we build them with a specific version but I feel like `:latest` is probably reasonable here rather than letting this get stale? Not a strong opinion though

Reply:
I am fine with `latest`. I don't think there is any precedence but this might actually give us better visibility when a new go version does break our build. I'll make the change!